### PR TITLE
Fixed typo in scripts/cspell/project-words.txt

### PR DESCRIPTION
**Pull request by priyanshsao**
## Which problem is this PR solving?
-  Part of #701 

## Description of the changes
-  Removed unnecessary words from the dictionary and updated their occurrences in the relevant files.
## How was this change tested?
- make spellcheck

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits

### CODE REVIEW

**Review by yurishkuro — COMMENTED** [review]
Thanks

File: scripts/cspell/project-words.txt [hunk 1]
```diff
@@ -36,8 +36,8 @@ hostnames
 hostport
 htpasswd
 ingester
-ingester's
 ingesters
+ingester's
```
Comment (yurishkuro) [inline scripts/cspell/project-words.txt:40]: Please revert order change

**Review by yurishkuro — CHANGES_REQUESTED** [review]

File: scripts/cspell/project-words.txt (hunk 1 above)
Comment (priyanshsao) [reply scripts/cspell/project-words.txt:40]: When the change is reverted and the make spellcheck command is run, the output displays a sorting error -

cat scripts/cspell/project-names.txt | grep -v '^#' | grep -v '^\s*$' | tr ' ' '\n' > scripts/cspell/project-names-parsed.txt
cd scripts/cspell && ./spellcheck.sh
sort: project-words.txt:40: disorder: ingesters
project-words.txt is not sorted.
make: *** [Makefile:60: spellcheck] Error 1

**Comment by yurishkuro** [comment]
It is failing now

**Comment by priyanshsao** [comment]
So, should I consider reverting the change?

**Comment by yurishkuro** [comment]
You need to make sure the linter passes. You can test it locally.

**Comment by priyanshsao** [comment]
The reason ingester's appears before ingesters in a sorted list is that it is treated as "ingester" followed by an apostrophe, whereas ingesters is treated as "ingester" followed by the letter s. This means that the apostrophe has a higher priority in the sort order than the letter s. If you were to place ingester's after ingesters, it would trigger a sorting error because the list is expected to follow alphabetical order, taking into account all characters, including punctuation. This is why it fails to pass the linter.

**Comment by yurishkuro** [comment]
I don't understand the motivation for your explanation - what are you trying to prove? Before your change the linter was not complaining, now it does complain.

**Comment by priyanshsao** [comment]
To check i ran the 'make checkspell'  in main branch and it showed the same error , i understand what you are saying but in main branch where nothing is changed still it is still  showing the error.

**Comment by priyanshsao** [comment]
@yurishkuro, I apologize for the confusion earlier. There was an issue with my terminal, which caused the error to appear.

**Pull request merged**